lib/api/client-commands/setGeolocation.js

```diff
@@ -0,0 +1,76 @@
+const ClientCommand = require('./_base-command.js');
+const {Logger} = require('../../utils');
+
+/**
+ * Mock the geolocation of the browser.
+ *
+ * @example
+ *  this.demoTest = function (browser) {
+ *    browser
+ *      // Set location of Tokyo, Japan
+ *      .setGeolocation({
+ *        latitude: 35.689487,
+ *        longitude: 139.691706,
+ *        accuracy: 100
+ *        })
+ *      .navigateTo('https://kawasaki-india.com/dealer-locator/');
+ *  };
+ *
+ * @method setGeolocation
+ * @syntax .setGeolocation({latitude, longitude, accuracy}, [callback])
```

Comment:
Is this the correct syntax or should it be `.setGeolocation(coordinates, [callback])`? If so, where should I specify what is expected in `coordinates` parameter?

Also, what would be `@api` here?

Reply:
Yeah the syntax is ok. I guess it's "User Prompts"?